Plugin/CartIntercept.php

```diff
@@ -159,20 +160,18 @@ public function _getItems(Cart $cart)
                     $_item['id'] = null;
                 }
                 if ($_item['id']) {
-                    $_item['qty'] = (int) $item->getQty();
+                    $_item['qty'] = (int)$item->getQty();
                     $_item['url'] = $item->getProduct()->getProductUrl();
-                    $_item['image']=$this->productHelper->getSmallImageUrl($product);
+                    $_item['image'] = $this->productHelper->getSmallImageUrl($product);
                     $current_price = null;
-                    $price_used = "reg";
```

Comment:
@lindsaycooper @silverman63 Same here and a few lines below here

Reply:
@aingram probably remnants of old implementation